### DISCUSSION TITLE
Allow users to create multiple dashboards

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9475] [Ideas]
> Created by: Ben Parizek (ben@barrelstrengthdesign.com) on 2015/03/12 23:19:32 +0000
> Votes at time of UserVoice import: 2

-----------------------------------

With the new charting API, there is a lot of potential for new items to be added to the dashboard.  With products such as Commerce, the dashboard also becomes a place where more types of data can exist: instructional notes to users, user data, quick post widgets, sales data.

While I'm not sure this feature is needed immediately, at some point, I feel it will be appropriate to allow admins to create multiple Dashboards and group similar widgets on their respective Dashboards.

For example, I might create a "Sales Dashboard" and a "Users Dashboard" and group all my Commerce widgets on the first, and User Stats on the latter.

**Comment by Djive** [comment]
any progress on this one..  now there is quiet many wigdets.. and this will be quite helpfull

**Comment by wmdhosting** [comment]
7 years soon :)  seems as good time to show some love to Dashboard :)
